src/java/org/apache/nutch/hostdb/UpdateHostDbReducer.java

```diff
@@ -74,6 +74,7 @@
     *
     * @param job
     */
+  @Override
```

Comment:
+1
This issue is the best example why @Override should be used: to avoid that by accident a method has not the intended signature.